pos_default_quantity/static/src/js/pos.js

```diff
@@ -0,0 +1,49 @@
+/*
+  Copyright 2019 Coop IT Easy SCRLfs
+    Robin Keunen <robin@coopiteasy.be>
+  License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+*/
+
+odoo.define(
+    'pos_default_quantity.pos_default_quantity',
+
+    function (require) {
+        "use strict";
+
+        var models = require('point_of_sale.models');
+
+        models.load_models({
```

Comment:
I'd add the default_qty field in the `product.uom` model instead.
If you want, it can be a related field to the `product.uom.categ`, for functional or usability reasons.

But technically speaking, the `product.uom` model is already loaded in the pos to get the `groupable` and `is_unit` fields:

https://github.com/odoo/odoo/blob/7132a15c38a3b5e5ec232404d21aacfa2ac5874e/addons/point_of_sale/static/src/js/models.js#L169-L182

So you could simply call [load_fields](https://github.com/odoo/odoo/blob/7132a15c38a3b5e5ec232404d21aacfa2ac5874e/addons/point_of_sale/static/src/js/models.js#L982) to load your extra field, and avoid loading yet another module to the pos ui: